scala/build/Dockerfile.maintenance

```
@@ -0,0 +1,266 @@
+	&& rm /home/qa/sbt.zip
+
+# Put tools like aws and sbt in the PATH
+ENV PATH /home/qa/.local/bin:/home/qa/sbt/bin:/home/qa/bin:${PATH}
+RUN sudo ln -s /home/qa/sbt/bin/sbt /usr/local/bin/sbt
+
+# aws-cli
+# RUN sudo curl -O https://bootstrap.pypa.io/get-pip.py \
+# 	&& python3 get-pip.py --user \
+# 	&& pip3 install awscli --upgrade --user \
+# 	&& sudo rm get-pip.py
+RUN pip install awscli --upgrade --user
```

Hadolint

[warning] Avoid use of cache directory with pip. Use `pip install --no-cache-dir <package>`

(DL3042)